examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -258,14 +252,16 @@ function vertical_diffusion_boundary_layer_cache(
 
     cond_type = NamedTuple{(:shf, :lhf, :E, :ρτxz, :ρτyz), NTuple{5, FT}}
 
+    normal = similar(Y.c, Geometry.WVector{FT})
```

Comment:
Actually, shouldn't this be `Geometry.WVector(ones(axes(Y.c)))`, because it's a unit vector?

Reply:
Ahh! That's probably why it was crashing, thank you!